content/200-concepts/300-more/300-telemetry.mdx

```diff
@@ -52,10 +52,9 @@ export CHECKPOINT_DISABLE=1
 
 ### Error reporting
 
-During the Preview period, data is potentially collected upon:
+Error data is potentially collected upon:
```

Comment:
```suggestion
Prisma potentially collects error data when there is a crash in the CLI.
```

Reply:
Also, does Prisma **always** collect error data when there is a crash in the CLI, or only sometimes? If it's always, we can lose "potentially".